api/environments/models.py

```diff
@@ -116,6 +117,11 @@ def get_from_cache(cls, api_key):
         except cls.DoesNotExist:
             logger.info("Environment with api_key %s does not exist" % api_key)
 
+    def get_feature_state(self, feature_id: int) -> typing.Optional[FeatureState]:
```

Comment:
Is this for unit tests only? why are we not using django filter?

Reply:
It is technically for the unit tests only, yes. You mean the package `django-filter` ? It's actually not something I've ever looked into (shame on me!). Do you think it would be useful here?

Reply:
Maybe we should add a comment? It's confusing since it returns the first feature state(out of many feature states for a feature in an environment)?

Reply:
Yes, that's a very good point. I get what you mean now, I'll update the code so that it's more obvious / correct.